database-plugins/docs/Database-action.md

```diff
@@ -0,0 +1,56 @@
+# Database Action
+
+
+Description
+-----------
+Action that runs a database script.
+
+
+Use Case
+--------
+The action can be used whenever you want to run a database script before or after a data pipeline.
+For example, you may want to run a sql update command on a database before the pipeline source pulls data from tables.
+
+
+Properties
+----------
+**scriptPath:** The file path of the database script to execute.
```

Comment:
huh? This runs a script on the local machine? So I need the script on every single machine in my cluster? I thought we talked about this, it should be almost identical to the QueryAction plugin.